### ISSUE TITLE
Google Chrome for mobile incorrectly identified as Safari

### ISSUE THREAD

**Issue by misterabd** [adobe/reactor-extension-core#32]
### Expected Behaviour
Using the Browser Condition and selecting Mobile Safari as an exclusion blocks only Mobile Safari on iOS.
### Actual Behaviour
Using the Browser Condition and selecting Mobile Safari as an exclusion blocks traffic from Chrome on Android as well. This is because Chrome reports 'Mobile Safari' in its [User Agent string](https://developer.chrome.com/multidevice/user-agent)
#### Sample Code that illustrates the problem
https://github.com/adobe/reactor-extension-core/blob/master/src/lib/conditions/helpers/clientInfo.js#L30
L30 needs to be swapped with L31 so that Chrome is "passed" first and that, indeed does allow Chrome for Android to work.

**Comment by thebenrob** [comment]
I've logged this internally as DTM-15864.

**Comment by thebenrob** [comment]
The DTM browser detection rules were pretty far out of date. We did update the browser detection rules when we released Launch in 2017, and they were up to date at the time. We haven't updated them since then. Mobile Safari was designed to match against Safari on iPhones and iPads (again, back in 2017). We have not checked it against any other iOS browsers. And Safari has made changes with Version 13 and 14 that introduced some strangeness, and at the moment I'm honestly not sure what "Mobile Safari" would match against.

We have discussed internally as a team. Our conclusion is that we will not fix this bug and will mark both the Browser and Operating System conditions as being "no longer supported" (as the Device Type condition already is).
Reasons:

User-agent detection of browsers and operating systems is a never-ending task that has always been tricky, our team is not staffed to build an automated way to test this against new browser versions as they come out or to do the effort manually. The most honest way to communicate this is to mark them as no longer supported.
Introducing a 3rd-party library to do the browser detection would add bloat to the Launch library. And none of the popular ones are very up-to-date anyway.
The Browser and Operating System conditions are in use less than 1000 times each across the entire Launch user base, representing less than 0.02% of all conditions. As a related data point, we usually drop runtime support for older browser versions when we notice that they drop below 0.1%.
So we won't delete them, and they'll continue to function as they do now, but we will not be fixing bugs that are logged against the functionality.
